test/conformance/README.md

```diff
@@ -9,3 +9,7 @@ this solution will no longer be necessary.
 When you fix any test, the match file must be updated
 Empty match files indicate that there are no failing tests
 in a particular group for the corresponding adapter.
+
+There is an argument NUMBER_OF_DEVICES that allow you to choose
```

Comment:
This section should be updated. Perhaps make a new subsection for this info or something

Also, please don't start the sentence with "There is an..." this sound like an intro to a fairy tale 😉

Reply:
Done